webpack/webpack.js

```diff
@@ -56,34 +56,43 @@ module.exports = env => ({
             utility: paths.src + '/utility/',
             react: paths.node_modules + '/react/',
             webrix: env.production ? paths.node_modules + '/webrix/' : paths.webrix,
-        }
+        },
+        fallback: {
+            assert: require.resolve('assert/'),
+            buffer: require.resolve('buffer/'),
+            fs: false,
+            module: false,
+            net: false,
+            path: require.resolve('path-browserify'),
+            process: require.resolve('process/browser'),
+        },
```

Comment:
These are imperative for `react-docgen` to be able to be imported correctly.

(*Took me hours to get this to work properly*)